UnitTests/CommonTestUtils/SubmoduleTestHelpers.cs

```diff
@@ -35,9 +35,22 @@ void ProviderStatusUpdated(object sender, SubmoduleStatusEventArgs e)
 
         public static async Task UpdateSubmoduleStatusAndWaitForResultAsync(ISubmoduleStatusProvider provider, GitModule module, IReadOnlyList<GitItemStatus> gitStatus)
         {
-            provider.UpdateSubmodulesStatus(workingDirectory: module.WorkingDir, gitStatus: gitStatus, forceUpdate: true);
+            List<DetailedSubmoduleInfo> result = new List<DetailedSubmoduleInfo>();
+            provider.StatusUpdated += Provider_StatusUpdated;
```

Comment:
Looks unfinished. Can be done but is kind of pointless with the empty handler.

Reply:
I'm not sure if it is needed at all...